src/components/study-pane.js

```diff
@@ -161,22 +160,16 @@ const StudyPane = ({ studyUuid, currentNode, setErrorMessage, ...props }) => {
     function renderMapView() {
         return (
             <ReactFlowProvider>
-                <div
-                    style={{
-                        display: 'flex',
-                        flexDirection: 'column',
-                        height: '100%',
-                    }}
-                >
-                    <div
-                        className={classes.horizontalToolbar}
+                <Box sx={styles.table}>
+                    <Box
+                        sx={styles.horizontalToolbar}
                         style={{
```

Comment:
why not putting thoses style rules in horizontalToolbar ?

Reply:
I initialy did not change this to stay focused on the JSS to emotion aspect of things, to minimize the impact of this PR. But it works and is cleaner. Ok for me.